app/pages/Help.js

```diff
@@ -19,56 +20,361 @@ class Help extends React.Component {
+              <li>Tap the Wi-Fi network you are currently connected to.</li>
+              <li>
+                Scroll down to <kbd>HTTP Proxy</kbd> and click Manual.
+              </li>
+              <li>
+                Enter your computer's IP address in the <kbd>Server</kbd> field.
+              </li>
+              <li>
+                Enter the port number in the <kbd>Port</kbd> field.
+              </li>
+              <li>
+                Leave Authentication <kbd>OFF</kbd>
+              </li>
+              <li>Save changes and exit settings</li>
+              <li>If you have Antivirus or a Firewall on your iOS, disable it</li>
+              <li>If you have a Firewall on your PC, disable it. Or allow SWEX trough by port.</li>
+              <li>Only if you use a VPN on your PC: Disable it. </li>
+            </ol>
+          </Accordion.Content>
+
+          {/* Troubleshooting */}
+          <Accordion.Title active={activeIndex === 6} index={6} onClick={this.handleAccordionClick.bind(this)}>
+            <Icon name="dropdown" />
+            Troubleshooting
+          </Accordion.Title>
+          <AccordionContent active={activeIndex === 6}>
+            Try those steps if you have issues with swex. Finding the issue :
+            <ol>
+              <li>Disable https mode in swex settings</li>
+              <li>Start proxy</li>
+              <li>Set proxy in phone / emulator</li>
+              <li>Start the game</li>
+              <li>Can you play it ?</li>
+              <li>Enable https mode</li>
+              <li>Start proxy</li>
+              <li>Set proxy in phone / emulator</li>
+              <li>
+                Start the game:
+                <ul>
+                  <li>
+                    I can play the game with https mode disabled:
+                    <ul>
+                      <li>
+                        I've got an error with https mode enabled:
+                        <ul>
+                          <li>Something is wrong with the certificate, open SWEX settings → regenerate cert → redo cert installation</li>
+                        </ul>
+                      </li>
+                    </ul>
+                  </li>
+                  <li>
+                    I can play the game with https enabled but i don't get any json:
+                    <ul>
+                      <li>Start SWEX as administrator</li>
+                      <li>Make sure the profile exporter is enabled in swex settings</li>
+                    </ul>
+                  </li>
+                  <li>
+                    I can't play the game with HTTPS mode disabled:
+                    <ul>
+                      <li>Are you connected to a network? → Does that network have access to the internet?</li>
+                    </ul>
+                  </li>
+                </ul>
+              </li>
+            </ol>
+          </AccordionContent>
+
+          {/* Common issues in setting up SWEX */}
```

Comment:
Not a huge fan of these meaningless comments, since you can see the section directly under it anyway. Would suggest to remove these.